CHANGELOG.md

```diff
@@ -5,63 +5,85 @@ CHANGELOG
-  - Rename all CloudFormation outputs from `Master*` to `HeadNode*`.
-  - Rename `NodeType` and tags from `Master` to `HeadNode`.
-- Remove `parallelcluster-` prefix from CloudFormation stack created by ParallelCluster.
+- Create a single ParallelCluster S3 bucket for each AWS region rather than for each cluster.
+- Adopt inclusive language
+  - Rename MasterServer to HeadNode in CLI outputs.
+  - Rename variable exported in the AWS Batch job environment from MASTER_IP to PCLUSTER_HEAD_NODE_IP.
+  - Rename all CFN outputs from Master* to HeadNode*.
+  - Rename NodeType and tags from Master to HeadNode.
+- Remove Ganglia support.
+- Add support for associating an existing Elastic IP to the head node.
+- Encrypt root EBS volumes and shared EBS volumes by default. Note that if the scheduler is AWS Batch, the root volumes
+  of the compute nodes cannot be encrypted by ParallelCluster.
 - Rename tags (Note: the following tags are crucial for ParallelCluster scaling logic):
   - `aws-parallelcluster-node-type` -> `parallelcluster:node-type`
   - `ClusterName` -> `parallelcluster:cluster-name`
   - `aws-parallelcluster-attributes` -> `parallelcluster:attributes`
   - `Version` -> `parallelcluster:version`
 - Remove tag: `Application`.
-- Prevent runtime baking, i.e. `pcluster create-cluster` only works for official AMIs or custom AMIs created by `pcluster build-image` command.
-- Retain CloudWatch logs on cluster deletion by default.
-- Remove Ganglia support.
-- Restrict access to Instance Metadata Service (IMDS) and add configuration parameter to manage it.
-- Remove instance store software encryption option (encrypted_ephemeral).
+- Remove [runtime creation method](https://docs.aws.amazon.com/parallelcluster/latest/ug/tutorials_02_ami_customization.html)
+  of custom ParallelCluster AMIs.
+- Retain CloudWatch logs on cluster deletion by default. If you want to delete the logs during cluster deletion, set
+  `Monitoring > Logs > CloudWatch > RetainOnDeletion` to False in the configuration file.
+- Remove instance store software encryption option (encrypted_ephemeral) and rely on default hardware encryption provided
+  by NVMe instance store volumes.
+- Add tag 'Name' to every shared storage with the value specified in the shared storage name config.
+- Slurm: Extend limits for supported number of queues to 10 and number of compute resources per queue to 5.
+- Upgrade Cinc Client to version 17.2.29.
 - Upgrade Slurm to version 20.11.8.
+- Upgrade NICE DCV to version 2021.1-10851.
 - Upgrade Cinc Client to version 17.2.29.
-- CLI commands do not default to `~/.parallelcluster/config` file anymore. The cluster configuration is now a required parameter.
-- Slurm: Extend limits for supported number of queues to 10 and number of compute resources per queue to 5.
+- Upgrade NVIDIA driver to version 470.57.02.
+- Upgrade CUDA library to version 11.4.0.
+- Remove installation of MPICH and FFTW packages.
+- Upgrade EFA installer to version 1.13.0
```

Comment:
Let's do all the  Upgrades first then all the Removes
Upgrade list: EFA, Slurm, DCV, ...[the rest]